config/initializers/rack_timeout.rb

```diff
@@ -1,15 +1,15 @@
 # frozen_string_literal: true
 
 # Adds a global timeout for all requests
-unless Rails.env.test?
+unless Rails.env.test? || !Settings.rack_timeout
```

Comment:
```suggestion
if !Rails.env.test? || Settings.rack_timeout
```

My puny brain can't process this logic. Can we invert to using `if` rather than `unless?

Reply:
@ericboehs Yeah you're right, that's much easier to read. Changed it